build-system/tasks/update-packages.js

```diff
@@ -101,6 +101,24 @@ function patchRegisterElement() {
   writeIfUpdated(patchedName, file);
 }
 
+/**
+ * Patch @ampproject/worker-dom/dist/index.safe.js with ES6 export.
+ */
+function patchWorkerDom() {
+  let file;
+  const patchedName =
+      'node_modules/@ampproject/worker-dom/dist/index.safe.patched.js';
+  file = fs.readFileSync(
+      'node_modules/@ampproject/worker-dom/dist/index.safe.js', 'utf8');
+  if (!/var MainThread=function\(R\){/.test(file)) {
+    throw new Error('Expected "var MainThread=function(R){" ' +
+      'to appear in @ampproject/worker-dom/dist/index.safe.js.');
+  }
+  file = file.replace('var MainThread=function(R){',
```

Comment:
This file needs a `replaceOrDir` utility to avoid the code duplication between match to see if it there and the replacement.